src/theories/Category.jl

```diff
@@ -200,21 +200,27 @@ abstract type TightExpr{T} <: GATExpr{T} end
 end
 
 """
-Theory of a pointed set-enriched category.
-We axiomatize a category equipped with zero morphisms.
+Theory of a pointed category.
+We axiomatize a category equipped with a zero object and zero morphisms.
+At the moment, the zero object is technically only required to be 
+a weak zero object; we don't ban nonzero maps to and from it.
 
 A functor from an ordinary category into a freely generated
-pointed-set enriched category, 
-equivalently, a pointed-set enriched category in which no two nonzero maps 
+pointed category, 
+equivalently, a pointed category in which no two nonzero maps 
 compose to a zero map, is a good notion
-of a functor that's total on objects and partial on morphisms.
+of a partial functor.
 """
-@theory ThPointedSetCategory{Ob,Hom} <: ThCategory{Ob,Hom} begin
+@theory ThPointedCategory{Ob,Hom} <: ThCategory{Ob,Hom} begin
+  ZeroOb()::Ob
```

Comment:
Our convention for term constructors in `Catlab.Theories` is that they are lowercase.